runtime/statemine/src/constants.rs

```diff
@@ -16,17 +16,17 @@
 pub mod currency {
 	use node_primitives::Balance;
 
-	/// The existential deposit. Set to 1/10 of its parent Relay Chain.
-	pub const EXISTENTIAL_DEPOSIT: Balance = 10 * CENTS;
+	/// The existential deposit. Set to 1/10 of its parent Relay Chain (v9020).
+	pub const EXISTENTIAL_DEPOSIT: Balance = CENTS / 10;
```

Comment:
I have had this idea for a long time: 

Very often while building mock runtimes, you want real values only for certain parameters. Due to hwo we manage cargo and overlapping dependencies, and a surge in compile-time, it is not desirable to pull in the `polkadot-runtime`/`kusama-runtime` itself. 

So I was thinking of pulling all the numeric values of each runtime into a dependency-free crate that can be easily imported in other projects. 

You could then also benefit from that here, and simply do: `kusama_runtime_config::EXISTENTIAL_DEPOSIT / 10` rather than this.